plugin/php-refactoring-toolbox.vim

```diff
@@ -70,6 +70,7 @@ if g:vim_php_refactoring_use_default_mapping == 1
     nnoremap <unique> <Leader>sg :call PhpCreateSettersAndGetters()<CR>
     nnoremap <unique> <Leader>cog :call PhpCreateGetters()<CR>
     nnoremap <unique> <Leader>da :call PhpDocAll()<CR>
+    nnoremap <unique> <C-r> :call Menu()<CR>
```

Comment:
`<c-r>` is redo in vim. Maybe you should supply a `<Plug>` mapping and then the user can map how they want?